pkg/config/config.go

```diff
@@ -59,6 +59,9 @@ type Server struct {
 
 	// currently only used in our demo app
 	HideApplications []string `def:"" desc:"please don't use, this will soon be deprecated"`
+
+	RetentionThresholdDays  int `def:"30" desc:"data retention threshold in days"`
+	RetentionThresholdDepth int `def:"3" desc:"data retention threshold in depth"`
```

Comment:
I don't think we need `RetentionThresholdDepth` option. I think `RetentionThresholdDays` might be enough.